tests/tt_eager/python_api_testing/sweep_tests/generation_funcs.py

```diff
@@ -1691,3 +1691,23 @@ def gen_ttnn_layernorm_args(
     mem_configs=[supported_mem_configs],
 ):
     return gen_dtype_layout_device(input_shapes, dtypes, layouts, mem_configs, do_sanitize_args=False)
+
+
+def gen_rand_exclude_range(size, range, low=0, high=100):
```

Comment:
Naming of variables can be more clear. Eg.   range -> exclude_range. lower -> exclude_lower etc.

Reply:
Fixed, variables renamed and boundary case problem resolved